### ISSUE TITLE
*** Net::IMAP::BadResponseError Exception: Unknown command a13mb109455439oti

### ISSUE THREAD

**Issue by mhluska** [gmailgem/gmail#252]
After running `gmail.inbox`, I'm getting the following error:

```
*** Net::IMAP::BadResponseError Exception: Unknown command a13mb109455439oti
```

I've tried:
- Enabling IMAP in gmail settings
- Allowing less secure apps in Account settings
- Using latest master with `gem 'gmail', git: 'https://github.com/gmailgem/gmail'`

The account I'm trying to pull mail from is a member of a G Suite account.

**Comment by jmtame** [comment]
I'm also seeing the same error

**Comment by Caesium20** [comment]
I'm also getting this error!  I see the past issues with this error saying this is a sporadic gmail error but i'm getting this quite consistently.

**Comment by Uysim** [comment]
Getting the same errors

**Comment by diegomelo182** [comment]
Hey dudes, this error occurs when the auth hasn't success, i solved this problem using IMAP OAuth 2.0 to get access token with oauth2.py tool, take a look on that and follow the steps:

https://github.com/nfo/gmail_xoauth#usage-for-oauth-20

**Comment by johnnyshields** [comment]
As of version 0.7.0 (Aug 19, 2018) this gem is officially deprecated and will no longer be maintained. Please instead use Google's official Gmail API Ruby Client, which uses the Gmail API rather than IMAP and has significantly better performance and reliability.
